src/mono/wasm/build/README.md

```diff
@@ -32,6 +32,8 @@ Implementation:
     - *after* any of the wasm build targets, use `AfterTargets="WasmBuildApp"` on that target
 - Avoid depending on this target, because it is available only when the workload is installed. Use `$(WasmNativeWorkload)` to check if it is installed.
 
+- `WasmEnableES6` will cause native re-link and produce `dotnet.js` as ES6 module.
```

Comment:
Expand on this to specify that in es6 mode nothing will appear in the global namespace by default

Reply:
We may also want to document somewhere (here?) that turning this on will break the debugger, and then we can remove that warning once we fix the debugger